# Simplied to have the same timestamp and gaze-idx for all three

uint8 status # TODO remove
string name
float32[3] pupil_center
float32 pupil_diameter
float32[3] gaze_direction

